astro-client/astro.go

```diff
@@ -177,7 +177,7 @@ func (c *HTTPClient) DeployImage(input ImageDeployInput) (*Image, error) {
 func (c *HTTPClient) ListClusters(vars map[string]interface{}) ([]Cluster, error) {
 	req := Request{
 		Query:     GetClusters,
-		Variables: map[string]interface{}{"input": vars},
+		Variables: map[string]interface{}{"organizationId": vars["organizationId"]},
```

Comment:
since we are only consuming `organizationId` out of the map, can we update the method to directly accept the `organizationId` instead of a map, will simplify the code.

`func (c *HTTPClient) ListClusters(organizationId string) ([]Cluster, error)`